app/assets/stylesheets/spotlight/_nestable.scss

```diff
@@ -99,15 +99,15 @@ tr.dd-item {
 
 .dd3-item > button { margin-left: 30px; }
 
-.dd3-handle { 
+.dd3-handle {
     position: absolute; margin: 0; left: 0; top: 0; cursor: pointer; width: 30px;
     text-indent: 100px;
     white-space: nowrap; overflow: hidden;
     border: 1px solid #aaa;
     background: #ddd;
-    background: -webkit-linear-gradient(top, #ddd 0%, #bbb 100%);
-    background:    -moz-linear-gradient(top, #ddd 0%, #bbb 100%);
-    background:         linear-gradient(top, #ddd 0%, #bbb 100%);
+    background: -webkit-linear-gradient(to bottom, #ddd 0%, #bbb 100%);
```

Comment:
Color literals like `#ddd` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#bbb` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Avoid vendor prefixes.